addon/components/g-map-polyline.js

```diff
@@ -78,11 +78,16 @@ const GMapPolylineComponent = Ember.Component.extend({
   setPath() {
     const polyline = this.get('polyline');
     const coordinates = this.get('coordinates');
+    const path = this.get('path');
 
     if (isPresent(polyline) && isPresent(coordinates)) {
       let coordArray = Ember.A(this.get('coordinates').mapBy('coordinate')).compact();
       polyline.setPath(coordArray);
     }
+
+    if (isPresent(polyline) && isPresent(path)) {
```

Comment:
And what if somebody sets both path and coordinates by mistake? Let's make at as `else if` option here. 

Or something like
```js
let path = this.get('path');

if (isEmpty(path) && isPresent(coordinates)) {
  path = Ember.A(this.get('coordinates').mapBy('coordinate')).compact();
}

if (isPresent(polyline) && isPresent(path)) {
  polyline.setPath(path);
}
```